packages/cli/src/lib/platform/authenticatedClient.spec.ts

```diff
@@ -102,4 +106,19 @@ describe('AuthenticatedClient', () => {
       AuthenticationStatus.LOGGED_IN
     );
   });
+
+  it('should create an API key with impersonate privileges', async () => {
```

Comment:
I think you should follow the naming convention used in the file, meaning:
```
it(`#funcName should...`)
```
So:
```suggestion
  it('#createImpersonateApiKey should create an API key with impersonate privileges', async () => {
```